src/helpers/build-image-service-url.js

```diff
@@ -1,12 +1,15 @@
 import qs from 'querystring';
 
+const capiRX = /^http:\/\/com\.ft\.imagepublish\.prod\.s3\.amazonaws\.com\/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/
```

Comment:
We just found this ourselves with the new image service, here is the PR we made -- https://github.com/Financial-Times/origami-image-service/pull/87